app/src/scripts/common/forms/array-input.jsx

```diff
@@ -84,6 +94,14 @@ class ArrayInput extends React.Component {
         <div className="text-info">{this.state.helper}</div>
         <div className="form-inline">
           <span>{inputFields}</span>
+          <ParamController
```

Comment:
This might work better if it was a higher level replacement for ArrayInput when dealing with parameters. We're making ArrayInput too general - better to simplify it for what it was originally for (editing the array values in datasets) and have another parameter component with simpler requirements.